src/runtime/rpc/network.cpp

```diff
@@ -42,6 +43,11 @@ namespace dsn {
 
 namespace security {
 DSN_DECLARE_bool(enable_auth);
+
+bool in_white_list(task_code code)
+{
+    return is_negotiation_message(code) || fd::is_failure_detector_message(code);
+}
```

Comment:
In my view, `in_white_list` is not a clear function name to show failure detector rpc needn't negotiation, I think it is better to use `is_negotiation_message(code) || fd::is_failure_detector_message(code)` directly in function `send_message` and `on_recv_message`, rather than the function `in_white_list`.

Reply:
I don't think so. White list means the rpc needn't negotiation.

Reply:
What about `in_negotiation_white_list`? I agree with @hycdong. The name doesn't point out the white list for what.

Reply:
The accurate  name is `in_security_white_list`, which means these rpcs doesn't need to do security process. 
But I think the name with namespace `security::in_white_list` is engough to represent it.

Reply:
I agree with @neverchanje , I think `in_negotiation_white_list` may be better, and `in_negotiation_allow_list` is much better.